aws/templates/docker_compose_visualizer.tftpl

```diff
@@ -9,4 +9,6 @@ services:
     ports:
       - 8050:8050
     volumes:
-      - ./logs/:/app/logs/
\ No newline at end of file
+      - ./logs/:/app/logs/
+    logging:
```

Comment:
is this needed to disable logging to json-file?

Reply:
yup, can also change docker deemon default log driver, but thought this is easier.